src/components/Dashboard/FilingHistoryList/filings/IncorporationApplication.vue

```diff
@@ -77,8 +78,8 @@ export default class IncorporationApplication extends Vue {
   @Prop({ required: true }) readonly filing!: ApiFilingIF
   @Prop({ required: true }) readonly index!: number
 
-  @Getter readonly getLegalName!: string
-  @Getter readonly getMyBusinessRegistryUrl!: string
+  @Getter(useBusinessStore) readonly getLegalName!: string
+  @Getter(useConfigurationStore) readonly getMyBusinessRegistryUrl!: string
```

Comment:
That's weird -- we don't usually mark getters as readonly. Please add a todo checkbox to the other ticket to fix this (here and elsewhere in this project).

Reply:
Added it to the cleanup ticket (https://github.com/bcgov/entity/issues/16207).